### PR TITLE
Support terraform file type

### PR DESCRIPTION
**Pull request by travisjeffery**
So you can limit searches to .tf files with --terraform.

Thanks.

### CODE REVIEW

File: tests/list_file_types.t [hunk 1]
```diff
@@ -321,6 +321,9 @@ Language types are output:
     --tcl
         .tcl  .itcl  .itk
   
+    --terraform
+        .tf
```
Comment (nielssorensen) [inline tests/list_file_types.t]: Can you also add .tfvars?

File: src/lang.c [hunk 2]
```diff
@@ -110,6 +110,7 @@ lang_spec_t langs[] = {
     { "stylus", { "styl" } },
     { "swift", { "swift" } },
     { "tcl", { "tcl", "itcl", "itk" } },
+    { "terraform", { "tf" } },
```
Comment (nielssorensen) [inline src/lang.c]: Add tfvars filetype support also.

**Comment by ggreer** [comment]
Please add support for .tfvars as well.

**Comment by travisjeffery** [comment]
Done

**Comment by travisjeffery** [comment]
@ggreer @nielssorensen

**Pull request merged**